.circleci/config.yml

```diff
@@ -44,49 +43,93 @@ jobs:
       - run:
           command: docker run -it -e TEST_BRANCH=${CIRCLE_BRANCH} edwardchee/scalyr-agent-ci:python26 /tmp/run_unittests.sh
 
-  smoke27:
+  smoke-standalone-27:
     docker:
       - image: circleci/python:2.7-jessie-browsers
     steps:
       - setup_remote_docker:
           docker_layer_caching: true
       - run:
-          command: docker run -it -e TEST_BRANCH=${CIRCLE_BRANCH} -e SCALYR_MAX_QUERY_SEC=300 -e PYTHON_VERSION=2.7 -e SCALYR_API_KEY=${SCALYR_API_KEY} -e READ_API_KEY=${READ_API_KEY} -e SCALYR_SERVER=${SCALYR_SERVER} scalyr/scalyr-agent-ci-smoketest /tmp/smoketest.sh
+          command: docker run -it -e TEST_BRANCH=${CIRCLE_BRANCH} -e MAX_WAIT=300 -e PYTHON_VERSION=2.7 -e SCALYR_API_KEY=${SCALYR_API_KEY} -e READ_API_KEY=${READ_API_KEY} -e SCALYR_SERVER=${SCALYR_SERVER} scalyr/scalyr-agent-ci-smoketest:v1.1 /tmp/smoketest_standalone.sh
```

Comment:
(discussion only) what versioning scheme are you using here?

Reply:
To be honest I didn't have time to put any thought into choosing a scheme, only that it should be something static, not `latest`, and something we can increment with different revisions.  Any preference/suggestions?